diff -uNr webCDwriter-2.7.2/configure webCDwriter-2.7.2/configure
--- webCDwriter-2.7.2/configure	2005-08-29 11:53:20.000000000 +0200
+++ webCDwriter-2.7.2/configure	2005-10-02 17:01:05.000000000 +0200
@@ -15,14 +15,14 @@
 # name. See the startup script $INITDIR/CDWserver.
 BINDIR=/usr/bin
 CONFIGDIR=/etc/CDWserver
-EXPORTDIR=/var/CDWserver/export
-HTTPDIR=/var/CDWserver/http
+EXPORTDIR=/var/lib/CDWserver/export
+HTTPDIR=/usr/share/CDWserver/http
 INITDIR=/etc/init.d
 LOGDIR=/var/log/CDWserver
-PROJECTSDIR=/var/CDWserver/projects
+PROJECTSDIR=/var/lib/CDWserver/projects
 SBINDIR=/usr/sbin
 SPOOLDIR=/var/spool/CDWserver
-TOOLSDIR=/var/CDWserver/bin
+TOOLSDIR=/usr/bin/CDWserver/bin
 
 # the default user and group that run CDWserver
 GROUP=cdwserve

diff -uNr webCDwriter-2.7.2/rcdrecord/configure webCDwriter-2.7.2/rcdrecord/configure
--- webCDwriter-2.7.2.orig/rcdrecord/configure	2005-08-29 11:56:59.000000000 +0200
+++ webCDwriter-2.7.2/rcdrecord/configure	2005-10-02 17:53:54.000000000 +0200
@@ -11,7 +11,7 @@
 
 # directories
 BINDIR=/usr/bin
-HTTPDIR=/var/CDWserver/http
+HTTPDIR=/usr/share/CDWserver/http
 
 # files
 CONFIGFILE=/etc/rcdrecord.conf
